.gitignore

```diff
@@ -1,4 +1,9 @@
 .DS_Store
 Thumbs.db
 docs/.vuepress/dist
-node_modules
\ No newline at end of file
+node_modules
+docs/it/extend/update-b8.md
```

Comment:
These (gitignore) changes are unnecessary; also, links to the upgrade guides need to be removed from the navbar

Reply:
how can i delete this file? (i'm new to github)

Reply:
No need to delete the files, just remove the added lines referencing .md files.

Reply:
i think it's done!